aiosip/contact.py

```diff
@@ -8,7 +8,7 @@
 
 
 # Regex pattern from p2p-sip project
-CONTACT_PATTERNS = [re.compile('^(?P<name>[a-zA-Z0-9\-\._\+~ \t]*)<(?P<uri>[^>]+)>(?:;(?P<params>[^\?]*))?'),
+CONTACT_PATTERNS = [re.compile('^(?P<name>["a-zA-Z0-9\-\._\+~ \t]*)<(?P<uri>[^>]+)>(?:;(?P<params>[^\?]*))?'),
```

Comment:
Could you provide me some examples that matches this regex ?
I think that this add should break some contacts examples.
It's definitely the time to have some unit tests for cases like that ;-)